### PR TITLE
Sql

### PR DESCRIPTION
**Pull request by gaelleDijon**

### CODE REVIEW

File: dags/sql/create-table.sql [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    tranche_age VARCHAR NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS regions(
+    id INTEGER UNIQUE NOT NULL PRIMARY KEY,
+    region_name VARCHAR(30)
+);
+
+CREATE TABLE IF NOT EXISTS departements(
+    num_dep VARCHAR(2) UNIQUE NOT NULL PRIMARY KEY,
```
Comment (charlotte-waegeneire) [inline dags/sql/create-table.sql]: :warning: `VARCHAR(3)` instead of 2 for DOMTOM

File: dags/sql/create-table.sql [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+--     colonne VARCHAR(10),
+--     metadata_type VARCHAR(30),
+--     description_fr VARCHAR(20),
+--     description_en VARCHAR(20),
+--     exemple VARCHAR(30)
+-- );   
+
+CREATE TABLE IF NOT EXISTS corona(
+    id SERIAL NOT NULL PRIMARY KEY,
+    departement VARCHAR(2) NOT NULL,
```
Comment (charlotte-waegeneire) [inline dags/sql/create-table.sql]: :warning: same, `VARCHAR(3)`

File: dags/sql/create-table.sql [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+CREATE TABLE IF NOT EXISTS regions(
+    id INTEGER UNIQUE NOT NULL PRIMARY KEY,
+    region_name VARCHAR(30)
+);
+
+CREATE TABLE IF NOT EXISTS departements(
+    num_dep VARCHAR(2) UNIQUE NOT NULL PRIMARY KEY,
+    dep_name VARCHAR(30),
+    id_region VARCHAR(10),
```
Comment (charlotte-waegeneire) [inline dags/sql/create-table.sql]: should be an `INTEGER` to be a foreign key

**Review by charlotte-waegeneire — CHANGES_REQUESTED** [review]
SQL query not working - check my comments and discord messages for further explanations :woman_dancing:

**Comment by gaelleDijon** [comment]
changes done

**Review by charlotte-waegeneire — APPROVED** [review]
perfect !

**Pull request merged**